lib/ejs.js

```diff
@@ -42,6 +42,7 @@ function resolveInclude(name, filename) {
   return includePath;
 }
 
+
```

Comment:
nit: extra new line

(if you are going to fix this, please rebase the commits. thanks.)